src/NuGetGallery.Core/Frameworks/NuGetFrameworkExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Frameworks;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace NuGetGallery.Frameworks
+{
+    public static class NuGetFrameworkExtensions
+    {
+        public static string GetBadgeVersion(this NuGetFramework framework)
+        {
+            var builder = new StringBuilder();
+            builder.Append($"{framework.Version.Major}.{framework.Version.Minor}");
```

Comment:
A bit of a premature optimization, but we can optimize this by replacing string formats with string builder appends. For example:

```suggestion
            builder.Append(framework.Version.Major);
            builder.Append(".");
            builder.Append(framework.Version.Minor);
```

Reply:
Updated.